client/reader/stream/x-post.jsx

```diff
@@ -80,9 +80,10 @@ class CrossPost extends PureComponent {
 
 	handleClick = ( event ) => {
 		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
-			return;
+			window.open( this.props.post.URL, '_blank', 'noopener noreferrer' );
```

Comment:
> in calypso? ha. ha. ha.

Some instincts you just can't shake :)

And my instinct says that including the `href` makes more sense semantically. It's also what we're doing in this scenario elsewhere in Calypso.